modin/engines/ray/utils.py

```diff
@@ -167,13 +167,7 @@ def initialize_ray(
                 "_redis_password": redis_password,
                 "logging_level": 100,
                 "_memory": object_store_memory,
-                "_lru_evict": True,
```

Comment:
Why did you remove this?

Reply:
That parameter is no longer supported in Ray init. That is why we had special casing for the code immediately below